### PR TITLE
Fix user_message type.

### PR DESCRIPTION
**Pull request by xuyanke**

### CODE REVIEW

File: Source/ObjectiveDropboxOfficial/Shared/Handwritten/Networking/DBRequestErrors.h [hunk 1]
```diff
@@ -11,6 +11,38 @@
...
+///
+- (instancetype)initWithText:(NSString *)text
+                      locale:(NSString *)locale;
+
+///
+/// Description method.
+///
+/// @return A human-readable representation of the current DBLocalizedUserMessage object.
+///
+- (NSString *)description;
```
Comment (northnose) [inline Source/ObjectiveDropboxOfficial/Shared/Handwritten/Networking/DBRequestErrors.h]: It looks like this is the same as the `text` property. `description` is already a property of `NSObject` so it seems unnecessary to redeclare it here.

File: Source/ObjectiveDropboxOfficial/Shared/Handwritten/Networking/DBRequestErrors.h [hunk 2]
```diff
@@ -11,6 +11,38 @@
...
 
+#pragma mark - Error Parameters
+
+/// A wrapper objct that has a localized human-readable error message and the locale.
+@interface DBLocalizedUserMessage : NSObject
+
+/// Localized, human-readable text.
+@property(nonatomic, readonly, copy) NSString *text;
+//  IETF BCP 47 language tag of text locale.
+@property(nonatomic, readonly, copy) NSString *locale;
```
Comment (northnose) [inline Source/ObjectiveDropboxOfficial/Shared/Handwritten/Networking/DBRequestErrors.h]: For consistency with the rest of the code, add a space after `@property`

File: Source/ObjectiveDropboxOfficial/Shared/Handwritten/Networking/DBRequestErrors.h [hunk 3]
```diff
@@ -32,8 +64,8 @@ NS_ASSUME_NONNULL_BEGIN
 /// the value of the "error_summary" key.
 @property (nonatomic, readonly, copy) NSString *errorContent;
 
-/// A human-readable error message that is optionally returned from some API endpoints.
-@property (nonatomic, readonly, copy, nullable) NSString *userMessage;
+/// An object contains human-readable error message that is optionally returned from some API endpoints.
```
Comment (northnose) [inline Source/ObjectiveDropboxOfficial/Shared/Handwritten/Networking/DBRequestErrors.h]: s/contains/containing a/

File: Source/ObjectiveDropboxOfficial/Shared/Handwritten/Networking/DBRequestErrors.h (hunk 1 above)
Comment (xuyanke) [reply Source/ObjectiveDropboxOfficial/Shared/Handwritten/Networking/DBRequestErrors.h]: yeah, it's unnecessary. I was trying to be consistent with the rest classes declared in this file. They all have `description` declared with the doc strings. Maybe people who worked this file wanted to have the explicit doc string for slightly better readability?

**Comment by xuyanke** [comment]
left a comment about `description`, will fix the whitespace and typo.

File: Source/ObjectiveDropboxOfficial/Shared/Handwritten/Networking/DBRequestErrors.h (hunk 1 above)
Comment (northnose) [reply Source/ObjectiveDropboxOfficial/Shared/Handwritten/Networking/DBRequestErrors.h]: Oh, I didn't realize it was like that elsewhere. SGTM.

**Comment by xuyanke** [comment]
update the pr to address whitespace and doc strings.

**Pull request merged**